File: sdk/csharp/libraries/microsoft.bot.solutions/Testing/DefaultTestAdapter.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

using Microsoft.Bot.Builder.Adapters;
using Microsoft.Bot.Solutions.Middleware;

namespace Microsoft.Bot.Solutions.Testing
{
    public class DefaultTestAdapter : TestAdapter
    {
        public DefaultTestAdapter()
            : base(sendTraceActivity: false)
        {
            Use(new EventDebuggerMiddleware());
        }
    }
}

File: sdk/typescript/libraries/bot-solutions/src/testing/defaultTestAdapter.ts
/**
 * Copyright(c) Microsoft Corporation.All rights reserved.
 * Licensed under the MIT License.
 */
import { TestAdapter } from 'botbuilder';
import { EventDebuggerMiddleware } from '../middleware';

export class DefaultTestAdapter extends TestAdapter {
    public constructor() {
        super(async (): Promise<void> => {});
        this.use(new EventDebuggerMiddleware());
    }
}
